### PR TITLE
docker-dind

### PR DESCRIPTION
**Pull request by subfuzion**
Convenience image based on latest Docker in Docker with experimental flag set and a few network utilities (like iproute2, dig and drill).

### CODE REVIEW

File: images/dind/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker build -t dind .
```
Comment (ndegory) [inline images/dind/build.sh]: shouldn't it be appcelerator/dind?

File: images/dind/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# dind
+
+[On Docker Hub](https://hub.docker.com/r/subfuzion/dind/). Cloned from [subfuzion/docker-dind](https://github.com/subfuzion/docker-dind).
```
Comment (ndegory) [inline images/dind/README.md]: we should create an appcelerator/dind repo on Docker hub

File: images/dind/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+     Go version:   go1.7.3
+     Git commit:   49bf474
+     Built:        Wed Jan 18 16:20:26 2017
+     OS/Arch:      linux/amd64
+     Experimental: true
+
+You can see from the last line that the daemon was started in experimental mode.
+With this image, however, experimental mode is the default:
+
+    $ docker run --detach --privileged --name=dind subfuzion:dind
```
Comment (ndegory) [inline images/dind/README.md]: and change the cmd to use appcelerator/dind instead of subfuzion:dind

**Review by ndegory — CHANGES_REQUESTED** [review]

File: images/dind/build.sh (hunk 1 above)
Comment (subfuzion) [reply images/dind/build.sh]: Yes, it was already removed, forgot to push.

File: images/dind/README.md (hunk 3 above)
Comment (subfuzion) [reply images/dind/README.md]: Yes, as discussed, will leave that up to you. Thanks.

**Review by ndegory — APPROVED** [review]

**Pull request merged**